src/components/AddActivityPage.js

```diff
@@ -1,26 +1,35 @@
 import React from 'react';
-import {connect} from 'react-redux';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
 import AcivityForm from './ActivityForm';
 import { addActivity } from '../actions/activities';
 import { ACTIVITY_PLAN_ROUTE } from '../constants/routes';
 
-const AddActivityPage = ({onAddActivity, history}) => (
+const AddActivityPage = ({ onAddActivity, history }) => (
   <div>
     <h1>Add Activity</h1>
     <AcivityForm
-      onSubmit={(activity) => {
+      onSubmit={activity => {
         onAddActivity(activity);
         history.push(ACTIVITY_PLAN_ROUTE);
       }}
     />
   </div>
 );
 
-const mapDispatchToProps = ({
+AddActivityPage.propTypes = {
+  onAddActivity: PropTypes.func,
+  history: PropTypes.object
+};
+
+const mapDispatchToProps = {
   onAddActivity: addActivity
-});
+};
 
-export default connect(null, mapDispatchToProps)(AddActivityPage);
+export default connect(
+  null,
+  mapDispatchToProps
+)(AddActivityPage);
 
 //TODO: Talk with Lukasz about comment:
```

Comment:
what is the problem ?

Reply:
nothing right now, as you probably noticed I have all of TODO's that you ever gave me in code right now, so this is just a note for me to ask you about this in next PR or maybe on call, just ignore those TODO's as I will keep in mind to remove those resolved in current PR